index.html

```diff
@@ -101,23 +101,124 @@ <h6>sas-ggdk</h6>
-		<p>Provides simple scripts and customization options to deploy monitoring, alerts, and log aggregation for Viya 4 running on Kubernetes.</p>
-		<p><a href="https://github.com/sassoftware/viya4-monitoring-kubernetes" title="viya4-monitoring-kubernetes">Contribute →</a></p>
+</div>
+
+<!-- Banded section -->
+
+<div class="divider">
+	<div>
+	<img class="u-img-divider" src="images/standard-section-divider.png">
 	</div>
-	<div class="boxed">
-		<h6>postgrest-client</h6>
-		<p>General purpose type-safe TypeScript client for PostgREST. Supports select, insert, update, upsert, delete queries with composite joins, resource embedding, full JSON column data handling and typing.
-		<p><a href="https://github.com/sassoftware/postgrest-client" title="postgrest-client">Contribute →</a></p>
+</div>
+
+<!-- Body section -->
+
+<div class="grid-container full" style="margin-bottom:3.3rem;">
+	<div>
+		<h2>Open source projects from SAS</h2>
+		<p>Browse the <a href="https://github.com/sassoftware" title="SAS Software organization on GitHub">SAS Software organization on GitHub</a>. Read the <a href="contributor-handbook.html" title="SAS Open Source Contributor Handbook">SAS Open Source Contributor Handbook</a> to get started.</p>
 	</div>
-	<div class="inverted">
-		<h3>Read the guide</h3>
-		<p>The SAS Open Source Contributor Handbook is your guide to contributing to SAS projects.</p>
-		<a class="button button-inverted" href="contributor-handbook.html" title="SAS Open Source Contributor Handbook">Read now</a>
+</div>
+
+<div class="grid-container halves">
+	<div id="just-launched">
+		<h3>Just launched</h3>
+	</div>
+	<div class="inverted" id="recently-updated">
+		<h3>Recently updated</h3>
+	</div>
+</div>
+	
+<!-- Banded section -->
+
+<div class="divider">
+	<div>
+	<img class="u-img-divider" src="images/standard-section-divider.png">
 	</div>
 </div>
 
+<!-- Body section -->
+
+<div class="grid-container halves" style="margin-bottom:5.3rem;">
+	<div>
+		<h2>SAS Contributes</h2>
+		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
+		<p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+	</div>
+	<div>
+		<img src="images/sas-contributes.png">
+	</div>
+</div>
+<div class="grid-container full">
+	<div>
+		<h3>Contributions from SAS</h3>
+		<p>Recent commits from <a href="https://github.com/orgs/sassoftware/people" title="Members of SAS Software organization">members of SAS Software</a>.
+		<hr>
+		<p class="u-monospace">{USER} did {ACTIVITY} on {PROJECT}.</p>
+		<p class="u-monospace">{USER} did {ACTIVITY} on {PROJECT}.</p>
+		<p class="u-monospace">{USER} did {ACTIVITY} on {PROJECT}.</p>
+		<hr>
```

Comment:
@isharacomix: This is the section where I'd like to have database data populated.